### PR TITLE
[Concept exercise] remote cars - guards/attrs

### PR DESCRIPTION
**Pull request by neenjaw**
Port of the Remote control car exercise.  Originally a C# exercise, but copied over from Elixir as source

### CODE REVIEW

File: exercises/concept/remote-control-car/remote_control_car_test.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    assert_equal "test_car", car.nickname
+  end
+
+  def test_required_key_nickname_should_not_have_default
+    assert_raises(ArgumentError) do
+      RemoteControlCar.new
+    end
+  end
+
+  def test_new_car_hasnt_been_driven
```
Comment (kotp) [inline exercises/concept/remote-control-car/remote_control_car_test.rb:22]:
```suggestion
  def test_new_car_has_not_been_driven
```

**Review by kotp — CHANGES_REQUESTED** [review]

**Comment by kytrinyx** [comment]
It's been 18 months since the last movement on this PR.

Does someone who was involved at the time have an idea of what it would take to get this over the finish line?

File: exercises/concept/remote-control-car/.docs/instructions.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# Instructions
+
+In this exercise you will be playing with a remote controlled car, which you have finally saved enough money to buy.
+
+Cars start with full (100%) batteries. Each time you drive the car using the remote control, it covers 20 meters and drains one percent of the battery.
```
Comment (kotp) [inline exercises/concept/remote-control-car/.docs/instructions.md:5]: Single sentence per line, as per Exercism's [markdown specification].

[markdown specification]: https://exercism.org/docs/building/markdown/markdown

**Review by kotp — COMMENTED** [review]
Draft, so a comment rather than request for change.

**Pull request closed without merging**